packages/controllers/dataset_detail.coffee

```diff
@@ -0,0 +1,17 @@
+if Meteor.isClient
+  Template.datasetDetail.onCreated ->
+    @subscribe('datasetDetail', @data.datasetId)
+
+  Template.datasetDetail.helpers
+    dataset: ->
+      Datasets.findOne(@datasetId)
+
+if Meteor.isServer
+  Meteor.publish 'datasetDetail', (id) ->
+    dataset = Datasets.findOne
+      _id: id
+      createdById: @userId
+    if dataset
+      [Datasets.find(dataset._id), RawFiles.find(dataset.file)]
```

Comment:
I don't like that this is accessing RawFiles, since otherwise it wouldn't need to be exported from models. I'm not sure how to handle it though - should the publication be in models?